text/rfcs/0006-metadata-in-forc-manifest.md

```diff
@@ -0,0 +1,96 @@
+- Feature Name: Metadata Section in Forc Manifest
+- Start Date: 2024-01-25
+- Author(s): @deekerno
+- RFC PR: [FuelLabs/rfcs#21](https://github.com/FuelLabs/rfcs/pull/21)
+
+## Summary
+
+An optional section describing metadata should be added to the Forc manifest.
+
+## Motivation
+
+The objective of this change is to allow a Sway contract developer to describe arbitrary information that will be available for Forc plugins to process. The immediate use of this change will be to allow for Sway contract developers to specify information for indexing the contract in the Forc manifest, if desired. The expected outcome is that developers will be able to define indexing-specific _metadata_ in the same place and manner that they define other important information, e.g. authoring information, dependencies, etc.
```

Comment:
```suggestion
The objective of this change is to allow a Sway developer to describe arbitrary information that will be available for Forc plugins to process. The immediate use of this change will be to allow for Sway contract developers to specify information for indexing the contract in the Forc manifest, if desired. The expected outcome is that developers will be able to define indexing-specific _metadata_ in the same place and manner that they define other important information, e.g. authoring information, dependencies, etc.
```

Reply:
i think we can remove contract and mention users as sway developers as they might not be developing contracts if we end up making sway a more general language that has indexing etc.

Reply:
I agree with that. 👍🏽